be/src/exec/pipeline/exchange/sink_buffer.cpp

```diff
@@ -265,7 +264,7 @@ void SinkBuffer::_try_to_send_rpc(const TUniqueId& instance_id) {
         }
 
         *request.params->mutable_finst_id() = _instance_id2finst_id[instance_id.lo];
-        request.params->set_sequence(_request_seqs[instance_id.lo]++);
```

Comment:
we would better add some comment for `_try_to_send_rpc`. Because we need to get the mutex before calling this function, otherwise we will get a concurrency problem.

Reply:
Ok, I'll add comments